src/Core/Resolver/OperationResolver.cs

```diff
@@ -39,12 +37,8 @@ public OperationResolver(ISnippets snippets, IWorkspace workspace, IReferences r
         private IEnumerable<AssemblyInfo> RelevantAssemblies()
         {
             if (snippets?.AssemblyInfo != null) yield return snippets.AssemblyInfo;
-            if (workspace?.AssemblyInfo != null) yield return workspace.AssemblyInfo;
-
-            foreach (var asm in references.Assemblies)
-            {
-                yield return asm;
-            }
+            foreach (var asm in workspace?.Assemblies) yield return asm;
```

Comment:
Completely tangential, so please disregard, but this just makes me wish C# had an equivalent to Python's `yield from` statement.

Reply:
Yes, I was looking for that...